controlplane/pkg/nsmd/constants.go

```diff
@@ -3,4 +3,5 @@ package nsmd
 const (
 	NsmServerSocketEnv = "NSM_SERVER_SOCKET"
 	NsmClientSocketEnv = "NSM_CLIENT_SOCKET"
+	WorkspaceEnv       = "WORKSPACE"
```

Comment:
Have a look at #515.  This doesn't have to be done in this patch, but keep it in mind.